ext/dom/tests/gh11404.phpt

```diff
@@ -111,27 +111,27 @@ var_dump($dom1->firstElementChild->firstElementChild->namespaceURI);
 -- Test append and attributes: with default namespace variation --
 NULL
 NULL
-string(0) ""
+NULL
 NULL
 string(7) "some:ns"
 <?xml version="1.0"?>
-<with xmlns="some:ns"><a/><b/><c xmlns=""/><d xmlns:x="some:ns" x:attrib="val"/><e attrib1="val" attrib2="val"/></with>
+<with xmlns="some:ns"><a xmlns=""/><b xmlns=""/><c xmlns=""/><d xmlns:x="some:ns" xmlns="" x:attrib="val"/><e attrib1="val" attrib2="val"/></with>
```

Comment:
Is it normal that the ``<d>`` tag here has 2 ``xmlns`` attributes?

Reply:
There's `xmlns:x` and `xmlns`. `xmlns:x` defines the namespace for prefix `x` which is used for `x:attrib`. The reason it also has `xmlns` is for the default namespace because `d` itself is in the empty namespace. If it were not to define `xmlns=""` then `d` would instead be in `some:ns` (inherited from `<with>`).
It's a tricky test case :)

Reply:
Right, thanks for the explanation!